app/stylesheets/main.scss

```diff
@@ -193,11 +193,6 @@ html {
     color: var(--sn-stylekit-info-color) !important;
 }
 
-ol *,
-li * {
-    font-family: inherit !important;
```

Comment:
There was a reason for this. Unfortunately I don't remember. Can you do more testing around why this might be essential?

Reply:
We could still use this, but without `!important`. This way we should be able to apply a font family

Reply:
(see video on the PR's description)